frontside/lib/routers/routers.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+import 'package:frontside/views/about/about.dart';
+import 'package:frontside/views/home/home.dart';
+
+
+
+class RouteManager{
+  static const String home = "/";
+  static const String about = "/about";
+
+  static Route<dynamic> generateRoute(RouteSettings routeSettings){
+
+    switch(routeSettings.name){
+      case home:
+        return MaterialPageRoute(
+          builder: (context) => Home(),
+
+        );
+      case about:
+        return MaterialPageRoute(
+            builder: (context) => About()
+
+        );
+
+      default:
+        throw FormatException("ups");
+
+    }
+
+  }
+
+}
```

Comment:
I see you're using the imperative approach for routing (`1.0`). I would rather go with `2.0` because this app is fresh new. You can read about the difference here: https://flutter.dev/docs/development/ui/navigation

I know that `2.0` comes with complexity. That's why I suggested using some lib like [beamer](https://pub.dev/packages/beamer) on top of it.

It'll give us some benefits, such as showing the correct URL in the address bar. For example, when you're on the `About` view, the URL address bar should be updated with `http://localhost:59803/about`, **which is not a case at this moment**:
<img width="400" alt="image" src="https://user-images.githubusercontent.com/85256521/123556597-5700de00-d78c-11eb-8b87-20e80a84f16b.png">

Could you refactor this to use `2.0` API, please? I don't want to force the `beamer` package, but I feel this is the best choice. There are two requirements:
- keep the code simple
- use `2.0` navigation